src/esn_input_layers.jl

```diff
@@ -118,3 +118,41 @@ function irrational_sign_input(res_size::Int,
     end
     return W_in
 end
+
+"""
+physics_informed_input(res_size::Int, in_size::Int, sigma::Float64, γ::Float64)
+
+Return a weighted input layer matrix, with random non-zero elements drawn from \$ [-\\text{sigma}, \\text{sigma}] \$, where some γ
+of reservoir nodes are connected exclusively to the raw inputs, and the rest to the outputs of the prior knowledge model , as described in [1].
```

Comment:
where is the model specified?

Reply:
This plan was to have it specified inside the ESN model itself (one of the arguments), so that it is used in both training and prediction as specified in the paper

Reply:
Yeha, I implemented a couple of ESN variations in the same manner just using a different the layer function in the general ESN constructor. At a quick glance the implementations is good, but I suppose we could wait for a quick comparison with the results of the paper before merging

Reply:
![Implementation](https://user-images.githubusercontent.com/73660335/97819315-c235f700-1c75-11eb-9fa0-2a3b209931be.jpg)

I hope I successfully captured their description. I have attached a screenshot of the part of the paper where they mention the implementation for the input layer. For the entire implementation to be complete still need to:

1. Introduce the prior model into an ESN
2. Do the same for the output weights as what was done in the input
3. make sure the training and prediction work properly

Reply:
I realised their implementation is to pair the input and knowledge model connections at random with a single reservoir - this isn't exactly the case in the PR - going to update this.